lib/MusicBrainz/Server/Controller/Rating.pm

```diff
@@ -33,8 +34,11 @@ sub rate : Local RequireAuth DenyWhenReadonly
     my $entity_id = $c->request->params->{entity_id};
     my $rating = $c->request->params->{rating};
 
-    unless (looks_like_number($rating)) {
-        $self->error( $c, message => 'rating must be a number', status => 400 );
+    unless (is_integer($rating) && $rating >= 0 && $rating <= 100) {
+        $c->stash(
+            message  => l('Rating should be an integer between 0 and 100'),
```

Comment:
sorry if pedantic, but if translating it I think we might as well use a full sentence.  (I see this message is just copied from rating_submit, but it's not translated there so doesn't really matter)
```suggestion
            message  => l('The rating should be an integer between 0 and 100.'),
```